### ISSUE TITLE
Observable collection without query should not fetch documents

### ISSUE THREAD

**Issue by 0x80** [0x80/firestore-mobx#1]
My intention is that when you set no query on a collection, it fetches no documents. This behaviour seems to have changed by accident. A collection without a query now fetches all documents.

**Comment by sallar** [comment]
@0x80 but isnt that the standard firestore behavior? no query will get all documents...

**Comment by 0x80** [comment]
@sallar True, but I found that behavior unpractical in combination with a mobx store, because sometimes at instantiation time you don't yet know what query you want to perform, but you might not want to start fetching all documents. 

The way firestore-mobx works you can set the collection name when instantiating the observable, so that it is locked to that collection/type, but then at a later point initiate the query for documents.

If you do want the default behavior of fetching all documents, you can still simply do that by setting a query with only an orderBy part, which I think makes sense anyway for those types of fetches.
